### ISSUE TITLE
Support for PostCSS / CSSNext

### ISSUE THREAD

**Issue by insanetisane** [pingyhq/pingy-cli#14]
It would be great to support either full access to PostCSS or even limited to CSSNext since that's the future of css essentially and eventually most developers will be using that syntax over Sass / Less.

**Comment by davej** [comment]
Yes, this is definitely under consideration. Config would possibly be done using something like this: https://www.npmjs.com/package/postcss-load-config

**Comment by insanetisane** [comment]
That looks good to me!

**Comment by randallmlough** [comment]
"Note: it's cssnext, not CSSNext" ;) 

BTW No jab intended - just thought it was funny that they put that on their site.

**Comment by insanetisane** [comment]
Oh haha I did zero research so not surprising I got it wrong...
